.travis.yml

```diff
@@ -8,10 +8,13 @@ jdk:
   - oraclejdk11
 
 env:
-  - SERVER_VERSION=wildfly1500 SECURITY_MGR=false BUILD_WFLY_MASTER=true
-  - SERVER_VERSION=wildfly1500,secmgr SECURITY_MGR=true BUILD_WFLY_MASTER=true
+  - SERVER_VERSION=wildfly1500,SECURITY_MGR=false BUILD_WFLY_MASTER=false
```

Comment:
this added comma seems wrong, env variable must be separated by space, see the wrong result at https://travis-ci.org/jbossws/jbossws-cxf/jobs/511870159#L453 and https://travis-ci.org/jbossws/jbossws-cxf/jobs/511870159#L8522